dist/example-config.js

```diff
@@ -193,12 +181,24 @@ var config = {
           }
         }
       },
-      dataSource: {
-        request: {
-          url: '//parsnip.msapp.co.nz/{x}/{y}'
-        },
-        type: 'singlePoll'
-      }
+      dataSource: 'parsnip'
+    }
+  ],
+  dataSources: [
+    {
+      name: 'geojson-spew',
```

Comment:
Any reason not to have dataSources an object keyed by name? I guess other than having to have quotes.